### DISCUSSION TITLE
Parallele running of Topas

### DISCUSSION THREAD

**Discussion by bcmohamed** [OpenTOPAS/OpenTOPAS#104] [Feature-Request] [answered]
Hi
How can I make the parallele running in the parameters file ?

**Comment by JoseRamosMendez** [comment, marked as answer]
Do you mean the use of [multithreaded simulations](https://opentopas.readthedocs.io/en/latest/parameters/overall/multithreading.html#multithreading) or navigation through [parallel worlds](https://opentopas.readthedocs.io/en/latest/parameters/geometry/parallel_world.html#parallel-worlds)? Please try [the user's guide](https://opentopas.readthedocs.io/en/latest/index.html), it has a good searching engine.
